components/button/floating-buttons.js

```diff
@@ -174,13 +171,19 @@ class FloatingButtons extends RtlMixin(LitElement) {
 
 		const viewBottom = window.innerHeight;
 		const containerRectHeight = this.getBoundingClientRect().height;
-		const containerTop = this._containerTop.getBoundingClientRect().top;
+		const containerTop = this.getBoundingClientRect().top;
```

Comment:
I think this should be equivalent.  

Do you know what the original purpose of this was?  I'm not 100% certain, but I think it _might_ have been from before this was a web component, and we needed to measure an element that was outside of the element being floated in order to determine whether to unfloat/float.

Reply:
I believe you are correct on the original purpose. When implementing this in Lit, I had to move `position: sticky` to be on the :host though rather than on the actual floating button content (the `div.d2l-floating-buttons-container` - it wasn't working in either safari or firefox https://github.com/BrightspaceUI/core/commit/08966be6edf95659473a9f69eb601ef54a66ee29#diff-47ef2563a3f8689f047bf8fc131e8363) so that made containerTop equivalent.